packages/iris-grid/src/IrisGridModel.ts

```diff
@@ -300,6 +300,10 @@ abstract class IrisGridModel<
 
   abstract set totalsConfig(totalsConfig: UITotalsTableConfig | null);
 
+  abstract isDeletableRange(range: GridRange): boolean;
+
+  abstract isDeletableRanges(ranges: readonly GridRange[]): boolean;
+
```

Comment:
Shouldn't need this anymore. `IrisGridModel` does not implement `DeletableGridModel` (or `EditableGridModel`).